ambry-api/src/main/java/com/github/ambry/config/ReplicationConfig.java

```diff
@@ -30,6 +30,11 @@ public class ReplicationConfig {
   public static final String REPLICATION_STANDBY_WAIT_TIMEOUT_TO_TRIGGER_CROSS_COLO_FETCH_SECONDS =
       "replication.standby.wait.timeout.to.trigger.cross.colo.fetch.seconds";
 
+  public static final String MAX_REPLICATION_RETRY_COUNT = "replication.retry.count.max";
+  public static final int DEFAULT_MAX_REPLICATION_RETRY_COUNT = 0;
+  @Config(MAX_REPLICATION_RETRY_COUNT)
+  public final int maxReplicationRetryCount;
```

Comment:
I think the property name `replication.retry.count.max` has to match to variable name. We convert the config property name to camel case form by removing the delimiter `.`. (Please see `convertToCamelCase` in closed source)

Reply:
fixed, good catch !